PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/nucleus/NucleusData.java

```diff
@@ -0,0 +1,311 @@
+
+            if (nickname.isPresent()) {
+                String nicknameString = HtmlUtils.swapColorsToSpan(TextSerializers.FORMATTING_CODE.serialize(nickname.get()));
+                inspectContainer.addValue("&nbsp;" + getWithIcon("Nickname", Icon.called("id-badge").of(Color.GREEN).of(Family.REGULAR)), nicknameString);
+            }
+        }
+
+        if (NucleusAPI.getNoteService().isPresent()) {
+            NucleusNoteService noteService = NucleusAPI.getNoteService().get();
+
+            List<Note> notes = noteService.getNotes(user);
+            if (!notes.isEmpty()) {
+                TableContainer notesTable = new TableContainer(
+                        getWithIcon("Noter", Icon.called("pen").of(Family.SOLID)),
+                        getWithIcon("Note", Icon.called("sticky-note").of(Family.REGULAR))
+                );
+
+                notesTable.setColor("light-blue");
+
+                for (Note note : notes) {
+                    String noter = "Unknown";
+                    if (note.getNoter().isPresent()) {
+                        User noterUser = getUser(note.getNoter().get());
+                        if (noterUser != null) {
+                            noter = noterUser.getName();
+                        }
+                    }
+
+                    notesTable.addRow(noter, note.getNote());
+                }
+
+                inspectContainer.addTable("Notes", notesTable);
+            }
+        }
+
+        if (NucleusAPI.getWarningService().isPresent()) {
+            NucleusWarningService warningService = NucleusAPI.getWarningService().get();
+
+            List<Warning> warnings = warningService.getWarnings(user);
+            inspectContainer.addValue(getWithIcon("Warning count", Icon.called("flag").of(Color.AMBER)), warnings.size());
+
+            if (!warnings.isEmpty()) {
+                TableContainer warningsTable = new TableContainer(
+                        getWithIcon("Warner", Icon.called("exclamation").of(Family.SOLID)),
+                        getWithIcon("Reason", Icon.called("sticky-note").of(Family.SOLID))
+                );
+
+                warningsTable.setColor("amber");
+
+                for (Warning warning : warnings) {
+                    String warner = "Unknown";
+                    if (warning.getWarner().isPresent()) {
+                        User warnerUser = getUser(warning.getWarner().get());
+                        if (warnerUser != null) {
+                            warner = warnerUser.getName();
+                        }
+                    }
+
+                    warningsTable.addRow(warner, warning.getReason());
+                }
+
+                inspectContainer.addTable("Warnings", warningsTable);
+            }
+        }
+
+        return inspectContainer;
+    }
+
+    @Override
+    public AnalysisContainer getServerData(Collection<UUID> uuids, AnalysisContainer analysisContainer) {
+        if (NucleusAPI.getWarpService().isPresent()) {
```

Comment:
This if block could be refactored into it's own method
`ifPresent(service -> newMethod(service, inspectContainer))`